tests/MongoDB.Driver.Core.Tests/Core/Clusters/ServerSelectors/CompositeServerSelectorTests.cs

```diff
@@ -71,5 +71,26 @@ public void Should_pass_on_the_filtered_servers_to_subsequent_selectors()
             mockSelector1.Verify(s => s.SelectServers(_description, _description.Servers), Times.Once);
             mockSelector2.Verify(s => s.SelectServers(_description, selector1Selected), Times.Once);
         }
+
+        [Fact]
+        public void Should_skip_subsequent_selectors_if_the_current_selector_returns_empty_servers_collection()
```

Comment:
Remove this test. It is testing code that I have requested be removed.

Reply:
Done.